<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cxbs</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="mtview" name="ui_gv"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>信息 cm</string>
       </attribute>
       <widget class="QToolBox" name="toolBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>380</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>380</width>
           <height>143</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <attribute name="label">
          <string>设置</string>
         </attribute>
         <widget class="QCheckBox" name="cb_shu">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>29</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>|</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cb_heng">
          <property name="geometry">
           <rect>
            <x>55</x>
            <y>10</y>
            <width>29</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>10</y>
            <width>48</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>优先级别</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>380</width>
           <height>143</height>
          </rect>
         </property>
         <attribute name="label">
          <string>柜体/变压器</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>24</y>
            <width>188</width>
            <height>77</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>宽度</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="le_gtwidth">
             <property name="placeholderText">
              <string>width</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>长度</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="le_gtdeep">
             <property name="placeholderText">
              <string>deep</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>380</width>
           <height>143</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="label">
          <string>间距</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>15</y>
            <width>212</width>
            <height>77</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="lal1">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;@1 移动左边的物体，请设置为负值。&lt;/p&gt;&lt;p&gt;@2 移动右边的物体，请设置为正值。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel#lal1{
color:red;
}</string>
             </property>
             <property name="text">
              <string>左右间距</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="le_hline"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lal2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;@1 移动上面的物体，请设置为负值。&lt;/p&gt;&lt;p&gt;@2 移动下面的物体，请设置为正值。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel#lal2{
color:red;
}</string>
             </property>
             <property name="text">
              <string>上下间距</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="le_vline">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pb_setjianju">
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QTextEdit" name="te_info">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>380</width>
          <height>180</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>变压器</string>
    </property>
    <addaction name="actionS11_30"/>
    <addaction name="actionS11_50"/>
    <addaction name="actionS11_63"/>
    <addaction name="actionS11_80"/>
    <addaction name="actionS11_100"/>
    <addaction name="actionS11_125"/>
    <addaction name="actionS11_160"/>
    <addaction name="actionS11_200"/>
    <addaction name="actionS11_250"/>
    <addaction name="actionS11_315"/>
    <addaction name="actionS11_400"/>
    <addaction name="actionS11_500"/>
    <addaction name="actionS11_630"/>
    <addaction name="actionS11_800"/>
    <addaction name="actionS11_1000"/>
    <addaction name="actionS11_1250"/>
    <addaction name="actionS11_1600"/>
    <addaction name="actionS11_2000"/>
    <addaction name="actionS11_2500"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>柜体</string>
    </property>
    <addaction name="action750_900"/>
    <addaction name="action900_1000"/>
    <addaction name="separator"/>
    <addaction name="action600_600"/>
    <addaction name="action800_600"/>
    <addaction name="action1000_600"/>
    <addaction name="action1200_600"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionXiFuGrid"/>
   <addaction name="separator"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="separator"/>
   <addaction name="actioncombine"/>
   <addaction name="separator"/>
   <addaction name="actionfenjie"/>
   <addaction name="separator"/>
   <addaction name="actionhjuzhong"/>
   <addaction name="separator"/>
   <addaction name="actionvjuzhong"/>
   <addaction name="separator"/>
   <addaction name="action_qingkong"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawGrid"/>
   <addaction name="actionword"/>
   <addaction name="actionbianyaqi"/>
   <addaction name="actionguiti"/>
   <addaction name="actionzhoupoint"/>
   <addaction name="actionlianxian"/>
  </widget>
  <action name="actionS11_30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-30</string>
   </property>
  </action>
  <action name="actionS11_50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-50</string>
   </property>
  </action>
  <action name="actionS11_63">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-63</string>
   </property>
  </action>
  <action name="actionS11_80">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-80</string>
   </property>
  </action>
  <action name="actionS11_100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-100</string>
   </property>
  </action>
  <action name="actionS11_125">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-125</string>
   </property>
  </action>
  <action name="actionS11_160">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-160</string>
   </property>
  </action>
  <action name="actionS11_200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-200</string>
   </property>
  </action>
  <action name="actionS11_250">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-250</string>
   </property>
  </action>
  <action name="actionS11_315">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-315</string>
   </property>
  </action>
  <action name="actionS11_400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-400</string>
   </property>
  </action>
  <action name="actionS11_500">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-500</string>
   </property>
  </action>
  <action name="actionS11_630">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-630</string>
   </property>
  </action>
  <action name="actionS11_800">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-800</string>
   </property>
  </action>
  <action name="actionS11_1000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-1000</string>
   </property>
  </action>
  <action name="actionS11_1250">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-1250</string>
   </property>
  </action>
  <action name="actionS11_1600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-1600</string>
   </property>
  </action>
  <action name="actionS11_2000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-2000</string>
   </property>
  </action>
  <action name="actionS11_2500">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S11-2500</string>
   </property>
  </action>
  <action name="action_qingkong">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actionDrawGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="asset.qrc">
     <normaloff>:/imgs/imgs/mao.png</normaloff>:/imgs/imgs/mao.png</iconset>
   </property>
   <property name="text">
    <string>DrawGrid</string>
   </property>
   <property name="toolTip">
    <string>网格</string>
   </property>
  </action>
  <action name="actionword">
   <property name="icon">
    <iconset resource="asset.qrc">
     <normaloff>:/imgs/imgs/t1.png</normaloff>
     <normalon>:/imgs/imgs/T.png</normalon>:/imgs/imgs/t1.png</iconset>
   </property>
   <property name="text">
    <string>word</string>
   </property>
   <property name="toolTip">
    <string>文字</string>
   </property>
  </action>
  <action name="actionXiFuGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>吸附</string>
   </property>
   <property name="toolTip">
    <string>吸附</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actionbianyaqi">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/imgs/b.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>变压器</string>
   </property>
   <property name="toolTip">
    <string>变压器</string>
   </property>
  </action>
  <action name="actionguiti">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/imgs/g.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>柜体</string>
   </property>
   <property name="toolTip">
    <string>柜体</string>
   </property>
  </action>
  <action name="action750_900">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>750*900</string>
   </property>
   <property name="toolTip">
    <string>750*900</string>
   </property>
  </action>
  <action name="action900_1000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>900*1000</string>
   </property>
  </action>
  <action name="actioncombine">
   <property name="text">
    <string>合并</string>
   </property>
   <property name="toolTip">
    <string>合并</string>
   </property>
  </action>
  <action name="actionfenjie">
   <property name="text">
    <string>分解</string>
   </property>
   <property name="toolTip">
    <string>分解</string>
   </property>
  </action>
  <action name="action600_600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>600*600</string>
   </property>
  </action>
  <action name="action800_600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>800*600</string>
   </property>
  </action>
  <action name="action1000_600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1000*600</string>
   </property>
  </action>
  <action name="action1200_600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1200*600</string>
   </property>
  </action>
  <action name="actionhjuzhong">
   <property name="text">
    <string>水平居中</string>
   </property>
   <property name="toolTip">
    <string>水平居中</string>
   </property>
  </action>
  <action name="actionvjuzhong">
   <property name="text">
    <string>垂直居中</string>
   </property>
   <property name="toolTip">
    <string>垂直居中</string>
   </property>
  </action>
  <action name="actionzhoupoint">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/imgs/point.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>周点</string>
   </property>
   <property name="toolTip">
    <string>周点</string>
   </property>
  </action>
  <action name="actionlianxian">
   <property name="icon">
    <iconset resource="asset.qrc">
     <normaloff>:/imgs/imgs/lianxian.png</normaloff>
     <normalon>imgs/lianxian.png</normalon>:/imgs/imgs/lianxian.png</iconset>
   </property>
   <property name="text">
    <string>连线</string>
   </property>
   <property name="toolTip">
    <string>连线</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mtview</class>
   <extends>QGraphicsView</extends>
   <header location="global">mtview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="asset.qrc"/>
 </resources>
 <connections/>
</ui>
